src/services/recim/team.ts

```diff
@@ -107,6 +111,13 @@ const deleteTeamParticipant = async (teamID: number, username: string) => {
 const editTeam = (ID: number, updatedTeam: PatchTeam): Promise<CreatedTeam> =>
   http.patch(`recim/Teams/${ID}`, updatedTeam);
 
+const getInvites = async (): Promise<Team[]> => http.get(`recim/Teams/invites`);
+
+const respondToTeamInvite = async (
+  teamID: number,
+  response: TeamInviteResponse,
+): Promise<CreatedTeamParticipant> => http.patch(`recim/Teams/${teamID}/invite`, response);
+
```

Comment:
```suggestion
const getTeamInvites = async (): Promise<Team[]> => http.get(`recim/Teams/invites`);

const respondToTeamInvite = async (
  teamID: number,
  response: TeamInviteResponse,
): Promise<CreatedTeamParticipant> => http.patch(`recim/Teams/${teamID}/invite`, response);

```

Reply:
Can we change the name to "getTeamInvites" so that it mirrors the "respondToTeamInvite", bit of a nitpick but it would be nice for consistency.